templates/nextflow-s3-template.yaml

```diff
@@ -98,37 +97,26 @@ Resources:
               - !Sub arn:aws:s3:::${Bucket}
               - !Sub arn:aws:s3:::${Bucket}/*
 
-          - !If
-            - NoReadWriteAccessArns
-            - Sid: ForgeUserReadWriteAccess
-              # gives forge user read/write write access to the bucket if no read/write arns are supplied
-              Effect: Allow
-              Principal:
-                AWS:
-                  - Fn::ImportValue: !Sub "${AWS::Region}-nextflow-forge-service-user-ServiceUserArn"
-              Action:
-                - "s3:*Object*"  
-              Condition:
-                StringEquals:
-                  s3:x-amz-acl: bucket-owner-full-control
-              Resource:
-                - !Sub arn:aws:s3:::${Bucket}/*   
-            - !Ref AWS::NoValue 
+          - Sid: ForgeUserFullAccess
+            # gives forge user full access to bucket
+            Effect: Allow
+            Principal:
+              AWS:
+                - Fn::ImportValue: !Sub "${AWS::Region}-nextflow-forge-service-user-ServiceUserArn"
+            Action:
+              - "s3:*"
```

Comment:
On second thought, we could stick to what's in the [Tower docs](https://github.com/seqeralabs/nf-tower-aws/blob/master/launch/s3-bucket-write.json) (copied below). This way, we would be more conservative with permissions. You can probably combine the bucket-specific and object-specific statements into one as you did. 

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket"
            ],
            "Resource": [
                "arn:aws:s3:::YOUR-BUCKET-NAME"
            ]
        },
        {
            "Action": [
                "s3:GetObject",
                "s3:PutObject",
                "s3:PutObjectTagging",
                "s3:DeleteObject"
            ],
            "Resource": [
                "arn:aws:s3:::YOUR-BUCKET-NAME/*"
            ],
            "Effect": "Allow"
        }
    ]
}
```